### PR TITLE
Fix Bug 1185471, improve layout of server sent error list

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

File: bedrock/mozorg/templates/mozorg/contribute/signup.html [hunk 1]
```diff
@@ -62,6 +45,16 @@ <h1 class="page-title">{{ _('To get started, tell us what interests you.') }}</h
             </li>
           {% endfor %}
         </ul>
+
+        {% if form.errors %}
+          {{ form.non_field_errors()|safe }}
+          <ul class="errorlist indented">
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/contribute/signup.html]: Looks a little odd indented here, is it for a reason?

**Comment by alexgibson** [comment]
One minor query, but otherwise this looks great! Let's squash these two commits :+1:

File: bedrock/mozorg/templates/mozorg/contribute/signup.html [hunk 2]
```diff
@@ -129,7 +122,11 @@ <h2 class="section-title">{{ _('Now tell us a little about yourself:') }}</h2>
           <li>
             <label for="id_email">{{ _('Email address') }}</label>
             {{ form.email }}
-            {{ form.email.errors }}
+            {% if form.email.errors %}
+              <ul class="errorlist">
+               <li>{{_('Please enter a valid email address.')}}</li>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/contribute/signup.html]: Nit: fix indentation

File: bedrock/mozorg/templates/mozorg/contribute/signup.html (hunk 1 above)
Comment (schalkneethling) [inline bedrock/mozorg/templates/mozorg/contribute/signup.html]: So, I have to ask ;p you mean the code indentation, or the way it is displayed in the UI?

Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/contribute/signup.html]: Bah, sorry for not being clear - I mean the way it is displayed in the UI. Seems a bit disjointed from the grid?

Comment (schalkneethling) [inline bedrock/mozorg/templates/mozorg/contribute/signup.html]: Yeah perhaps. The aim was for it to line up better with the icons above but, perhaps it is best to just leave it left aligned?

Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/contribute/signup.html]: Up to you I guess, but seems a bit neater left aligned to me.

**Comment by schalkneethling** [comment]
Updated and squished :octopus:

**Pull request merged**